math/minuit2/src/MnMachinePrecision.cxx

```diff
@@ -9,25 +9,35 @@
 
 #include "Minuit2/MnMachinePrecision.h"
 #include "Minuit2/MnTiny.h"
+#include <limits>
 
 namespace ROOT {
 
    namespace Minuit2 {
 
 
-MnMachinePrecision::MnMachinePrecision() :
-   fEpsMac(4.0E-7),
-   fEpsMa2(2.*sqrt(4.0E-7)) {
+MnMachinePrecision::MnMachinePrecision() {
+   // use double precision values from the numeric_limits standard
+   // and do not determine it anymore using ComputePrecision
+   // epsilon from stundard
+   // note that there is a factor of 2 in the definition of
+   // std::numeric_limitys::epsilon w.r.t DLAMCH epsilon
+   
+   fEpsMac = 4. * std::numeric_limits<double>::epsilon();
+   fEpsMa2 = 2.*sqrt(fEpsMac);
+}
+void MnMachinePrecision::ComputePrecision() {
```

Comment:
Is this used anywhere?

Reply:
No it is not used, but I still would prefer to leave this code for the time being. 

Thank you for the comment on copy/ctor and assignment. One can of course use default ones